R/LearnerClassifGAMBoost.R

```diff
@@ -0,0 +1,99 @@
+#' @title Boosted Generalized Additive Classification Learner
+#'
+#' @aliases mlr_learners_classif.gamboost
+#' @format [R6::R6Class] inheriting from [LearnerClassif].
+#'
+#' @description
+#' A [LearnerClassif] for a classification gamboost implemented in [mboost::gamboost()] in package \CRANpkg{mboost}.
```

Comment:
Don't you need to link with `[mlr3::LearnerClassif]`?

Reply:
Works without it but changed it for clarity